feature/policy_forwarding/policy_vrf_selection/otg_tests/policy_based_traffic_steering_default_to_nondefault/README.md

```diff
@@ -0,0 +1,174 @@
+		- IPv4Prefix6/24 to IPv4Prefix1/24 
+		- IPv4Prefix7/24 to IPv4Prefix2/24
+		- IPv4Prefix8/24 to IPv4Prefix3/24 
+		- IPv4Prefix9/24 to IPv4Prefix4/24 
+		- IPv4Prefix10/24 to IPv4Prefix5/24
+		- IPv6Prefix6/64 to IPv6Prefix1/64 
+		- IPv6Prefix7/64 to IPv6Prefix2/64
+		- IPv6Prefix8/64 to IPv6Prefix3/64 
+		- IPv6Prefix9/64 to IPv6Prefix4/64 
+		- IPv6Prefix10/64 to IPv6Prefix5/64    
+	- Expectations:
+		- All traffic must be successful<br><br><br>
+
+### PF-1.6.2: Traffic from ATE2:Port1 to ATE1:Port1 Prefix 1 migrated to Non-Default VRF using the VRF selection policy
+  * ATE2 sends following IPv4 and IPv6 flows:
+    * IPv4Prefix6/24 to IPv4Prefix1/24 
+    * IPv4Prefix7/24 to IPv4Prefix2/24
+    * IPv4Prefix8/24 to IPv4Prefix3/24 
+    * IPv4Prefix9/24 to IPv4Prefix4/24 
+    * IPv4Prefix10/24 to IPv4Prefix5/24
+    * IPv6Prefix6/64 to IPv6Prefix1/64 
+    * IPv6Prefix7/64 to IPv6Prefix2/64
+    * IPv6Prefix8/64 to IPv6Prefix3/64 
+    * IPv6Prefix9/64 to IPv6Prefix4/64 
+    * IPv6Prefix10/64 to IPv6Prefix5/64
+  * DUT has the following VRF selection policy
+    * Statement1: traffic matching IPv4Prefix1/24 & IPv6Prefix1/64, Punt to Non-default vrf
+    * Statement2: traffic matching IPv4Prefix2/24 & IPv6Prefix2/64, Punt to default vrf
+    * Statement3: traffic matching IPv4Prefix3/24 & IPv6Prefix3/64, Punt to default vrf
+    * Statement4: traffic matching IPv4Prefix4/24 & IPv6Prefix4/64, Punt to default vrf
+    * Statement5: traffic matching IPv4Prefix5/24 & IPv6Prefix5/64, Punt to default vrf
+    * Statement6: traffic matching IPv4Prefix6/24 & IPv6Prefix6/64, Punt to default vrf
+  * DUT must also leak all the routes from the Default VRF to the non-default VRF
+  * Expectations:
+    * Traffic for Prefix 1 received from ATE2:Port1 once punted to non-defailt VRF by the VRF selection policy, must be received by ATE1:Port1
+    * Traffic sent by ATE2:Port1 must be routed to ATE1:Port1 via the DEFAULT VRF in the DUT.<br><br><br>
+
+**PF-1.6.3 to PF-1.6.7: Traffic from ATE2:Port1 to ATE1:Port1 migrated to Non-Default VRF using the VRF selection policy.**
+Follow the steps in PF-1.6.2 above to gradually move different traffic flows from the Default VRF to the Non-Defailt in the following sequence:
+
+  * PF-1.6.3 Prefix 1-2 Traffic from ATE2:Port1 to ATE1:Port1 migrated to Non-Default VRF using the VRF selection policy.
+    VRF selection policy on DUT:Port2 changes as follows:
+    * Statement1: traffic matching IPv4Prefix1/24 & IPv6Prefix1/64, Punt to Non-default vrf
+    * Statement2: traffic matching IPv4Prefix2/24 & IPv6Prefix2/64, Punt to Non-default vrf
+    * Statement3: traffic matching IPv4Prefix3/24 & IPv6Prefix3/64, Punt to default vrf
+    * Statement4: traffic matching IPv4Prefix4/24 & IPv6Prefix4/64, Punt to default vrf
+    * Statement5: traffic matching IPv4Prefix5/24 & IPv6Prefix5/64, Punt to default vrf
+    * Statement6: traffic matching IPv4Prefix6/24 & IPv6Prefix6/64, Punt to default vrf<br><br>
```

Comment:
Here any below, you could both simplify and clarify this to:

* Modify the DUT generated config to change  vrf selecion policy Statement2 to:
-  traffic matching IPv4Prefix2/24 & IPv6Prefix2/64, Punt to Non-default vrf
* Use gnmi.Set REPLACE to push the config to the DUT

I think this is how we would want the code to be written to be easier to read and to ensure we are using gnmi.Set REPLACE option with the full policy-forwarding 'configlet'.  This is closer to the operational use case.

Reply:
done , updated for each statement